app/src/main/java/org/fossasia/pslab/fragment/LuxMeterFragmentConfig.java

```diff
@@ -157,8 +160,23 @@ else if (value < updatePeriodMin)
             @Override
             public void onItemSelected(AdapterView<?> parent, View view, int position, long id) {
                 try {
-                    if (bh1750 != null) {
-                        bh1750.setRange(gainValue.getSelectedItem().toString());
+                    switch (position) {
+                        case 0:
```

Comment:
What are these blank case statements?

Reply:
For all cases 0, 1 and 2 do the same thing